odoo/addons/test_main_flows/static/tests/tours/main_flow.js

```diff
@@ -726,7 +726,7 @@ tour.register('main_flow_tour', {
     trigger:  ".o_field_widget[name=balance_end_real] input",
     content: _t('Let\'s calculate the ending balance.'),
     position: 'right',
-    run: 'text 9010.85', // + 12.65
+    run: 'text 9955.87',
```

Comment:
gnii?

Reply:
The bank statement created in demo is posted directly. However, you can only post statement having `balance_end == balance_end_real` that is not the case of the actual data or the one created by the tour.